### PR TITLE
Make PA 40% less jank

### PR DESCRIPTION
**Pull request by metalgearsloth**
I just blocked the buttons until the state comes in as I was too lazy to fix spam clicking properly. The main issue is the engine problem + the fact that the console can't reach level 4 anyway.

:cl:
- fix: PA controls now much less jank.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml [hunk 1]
```diff
@@ -177,6 +177,7 @@
       - Impassable
       - Opaque
   - type: Projectile
+    soundHit:
```
Comment (PaulRitter) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml:180]: maybe like this?
```suggestion
    soundHit: null
```

**Comment by wrexbe** [comment]
It has some weird bugs now.

- If I assemble the computer (without assembling the rest of it), I see all the buttons are not disabled, and if I click stength+, the strength goes to 1, and then all the buttons disable.

- If I assemble the computer (without assembling the rest of it), and click ON, the server crashes ( on debug)
```
 unhandled: Robust.Shared.Utility.DebugAssertException: Exception of type 'Robust.Shared.Utility.DebugAssertException' was thrown.
   at Robust.Shared.Utility.DebugTools.Assert(Boolean condition) in C:\Users\Wrexbe2\RiderProjects\space-station-14\RobustToolbox\Robust.Shared\Utility\Debu
gTools.cs:line 32
   at Content.Server.ParticleAccelerator.Components.ParticleAcceleratorControlBoxComponent.SwitchOn() in C:\Users\Wrexbe2\RiderProjects\space-station-14\Con
tent.Server\ParticleAccelerator\Components\ParticleAcceleratorControlBoxComponent.cs:line 444
```

**Review by wrexbe — DISMISSED** [review]
It has some weird bugs now.

- If I assemble the computer (without assembling the rest of it), I see all the buttons are not disabled, and if I click stength+, the strength goes to 1, and then all the buttons disable.

- If I assemble the computer (without assembling the rest of it), and click ON, the server crashes ( on debug)
```
 unhandled: Robust.Shared.Utility.DebugAssertException: Exception of type 'Robust.Shared.Utility.DebugAssertException' was thrown.
   at Robust.Shared.Utility.DebugTools.Assert(Boolean condition) in C:\Users\Wrexbe2\RiderProjects\space-station-14\RobustToolbox\Robust.Shared\Utility\Debu
gTools.cs:line 32
   at Content.Server.ParticleAccelerator.Components.ParticleAcceleratorControlBoxComponent.SwitchOn() in C:\Users\Wrexbe2\RiderProjects\space-station-14\Con
tent.Server\ParticleAccelerator\Components\ParticleAcceleratorControlBoxComponent.cs:line 444
```

**Comment by wrexbe** [comment]
If you want this to be null, I think you need to change `[DataField("soundHit", required: true)]` to not be required, also I think the code that is using it, is assuming it's not null. `component.SoundHit.GetSound()`

**Comment by metalgearsloth** [comment]
> If you want this to be null, I think you need to change `[DataField("soundHit", required: true)]` to not be required, also I think the code that is using it, is assuming it's not null. `component.SoundHit.GetSound()`

The yaml parsing was erroring which needed fixing first.

**Pull request merged**